DOCSP-35948 write operations - insert documents (#2804)
jenssegers/laravel-mongodb: docs/fundamentals.txt

.. _laravel_fundamentals:

============
Fundamentals
============

.. facet::
   :name: genre
   :values: reference

.. meta::
   :keywords: php framework, odm

.. toctree::
   :titlesonly:
   :maxdepth: 1

   /fundamentals/read-operations
   /fundamentals/write-operations

> Learn how to perform the following tasks by using the {+odm-long+} in the
> following pages:

Reviewer: S: this sentence is a little repetitive; you could reword to something like:
```suggestion
Learn how to use the {+odm-long+} to perform the following tasks:
```

Author: Thanks, I like this reworded version!
